### ISSUE TITLE
Raft partition continues to throw exceptions after healing from network partition 

### ISSUE THREAD

**Issue by menski** [zeebe-io/zeebe#2602]
After creating a network partition in a 5 node cluster and restoring the healthy network the isolated nodes continues to log the following exceptions:

```
12:50:27.396 [] [raft-server-system-partition-1] WARN  io.atomix.protocols.raft.roles.FollowerRole - RaftServer{system-partition-1}{role=FOLLOWER} - java.net.ConnectException
12:50:28.085 [] [raft-server-raft-atomix-partition-3] WARN  io.atomix.protocols.raft.roles.FollowerRole - RaftServer{raft-atomix-partition-3}{role=FOLLOWER} - java.net.ConnectException
12:50:30.715 [] [raft-server-raft-atomix-partition-3] WARN  io.atomix.protocols.raft.roles.FollowerRole - RaftServer{raft-atomix-partition-3}{role=FOLLOWER} - java.net.ConnectException
```

It seems to be related to atomix raft partitions and not the distributed log. The cluster seems to be healthy and all partitions can create and complete workflow instances.

**Comment by npepinpe** [comment]
I don't see this happening again; has this happened to anyone since this was opened? It's hard to tell what's going on without being able to reproduce it. I've been working on the two other bugs where I restart and/or partition a lot, and none of the 3 bugs is happening.

**Comment by npepinpe** [comment]
Related to https://github.com/zeebe-io/atomix/issues/16

Fix for now is to add all nodes as initial contact points.

**Comment by npepinpe** [comment]
Related to #2684 (should be closed when we close that one)

**Comment by npepinpe** [comment]
Closed by #2756
